protocol/protobuf/sync_settings.proto

```diff
@@ -31,8 +31,7 @@ message SyncSetting {
     DISPLAY_NAME = 13;
     BIO = 14;
     MNEMONIC_REMOVED = 15;
-    ENS_USERNAMES = 16;
-    INCLUDE_WATCHONLY_ACCOUNT = 17;
+    INCLUDE_WATCHONLY_ACCOUNT = 16;
```

Comment:
hmm, shouldn't we keep it `17`?

Reply:
that's not quite it, otherwise it gets crazy, we test things against desktop on mobile to make sure releases are compatible, especially syncing, we need to keep some level of compatibility, otherwise we spend to much time investigating bugs that might be bugs or are just breaking changes. 
Avoid breaking changes when necessary, even if we are pre-mvp (if the change wasn't released etc, there's no issue, but if it was released, and it's a minor change like this, seems very unnecessary to break compatibility).

Reply:
And in this case it's actually a very straight, simple and right way of making things compatible.
This explicit protobuf nubmering was created for exactly such case 🙂

Reply:
thank you guys! I heard more voices support my old view now, i should insist it 😅 fixed